src/recoilState/app.ts

```diff
@@ -398,3 +399,74 @@ export const useProfile = (address: string) =>
 
 export const useEpochsStatus = (circleId: number) =>
   useRecoilValue(rCircleEpochsStatus(circleId));
+
+interface ICircleTask {
+  id: string;
+  name: string;
+  permalink: string;
+  assignees: {
+    id: string;
+    threepids: {
+      source: string;
+      threepid: string;
+    }[];
+  }[];
+}
+
+export const rSelectedCircleTasks = selector<ICircleTask[]>({
+  key: 'rSelectedCircleTasks',
+  get: async ({ get }) => {
+    const circle = get(rSelectedCircle);
+    if (!circle.circle.dework_organization_id) return [];
+    return fetch(`${DEWORK_API_URL}/graphql`, {
```

Comment:
Curious about error handling and if the API is down.

Reply:
In the long run we would like to do:
https://hasura.io/docs/latest/graphql/core/remote-schemas/index.html